tests/ut/test_gpu_search.cc

```diff
@@ -45,20 +47,29 @@ TEST_CASE("Test All GPU Index", "[search]") {
         return json;
     };
 
-    auto ivfsq_gen = ivfflat_gen;
-
     auto ivfpq_gen = [&ivfflat_gen]() {
         knowhere::Json json = ivfflat_gen();
         json[knowhere::indexparam::M] = 0;
         json[knowhere::indexparam::NBITS] = 8;
         return json;
```

Comment:
please add a test for a laaaaaaarge IVFPQ topk value somewhere

Reply:
ok，i will  test it.